bin/ResumeChecker.py

```diff
@@ -227,6 +227,41 @@ def term_match(string_to_search, term):
                       str(string_to_search) + ': ' + str(exception_instance))
         return None
 
+def check_title_city_state(string_to_search):
+    """
+        Find the title of the job, location by city and state
+        :param string_to_search: A string to check for a physical address in
+        :type string_to_search: str
+        :return: array of strings with job, location by city and state
+        :rtype: bytearray
+        """
+    try:
+        regular_expression = re.compile(r"([A-Z][a-z][\w-]*(\s+[A-Z][\w-]*)+),\s([^,]+),\s([A-Z]{2})")
+        result = re.findall((regular_expression), string_to_search)
+
+        if len(result)>0:
+            #print len(result)
```

Comment:
If this is non-running code, it likely should be removed. Or, if the number of results is interesting, it could be output with `logging.debug('Number of results: {}'.format(len(result)))` or similar